code/src/components/Form.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+
+import { Summary } from './Summary'
+
+const colors = ['red wine', 'white wine', 'rosé wine']
+
+export const Form = () => {
+  const [name, setName] = useState('')
+  const [wineColor, setWineColor] = useState('')
+  const [country, setCountry] = useState('')
+  const [counter, setCounter] = useState(0)
+
+  const onNameChanged = (event) => {
+    console.log(event.target.value)
+    setName(event.target.value)
+  }
+  const onWineColorChanged = (event) => {
+    console.log(event.target.value)
+    setWineColor(event.target.value)
+  }
+  const onCountryChanged = (event) => {
+    console.log(event.target.value)
+    setCountry(event.target.value)
+  }
+  const onCounterChanged = (event) => {
+    event.preventDefault()
+    setCounter(counter + 1)
```

Comment:
Smart and short (and we like short code! 🙂) way you did with the hide and show of summary. 👌